### PR TITLE
Reimplement vue websocket support.

### PR DESCRIPTION
**Pull request by mshima**
This was meant to be a fix for unhandled exceptions, but in the end it became a full rewrite.

- Switches from deprecated/archived webstomp-client to `@stomp/rx-stomp`.
- Switch from account service doing the connect/disconnect to Vuex store watch.

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/client/templates/vue/src/main/webapp/app/admin/tracker/tracker.service.ts.ejs [hunk 1]
```diff
@@ -1,107 +1,114 @@
...
 <%_ } _%>
-    this.stompClient.connect(headers, () => this.afterConnect());
+    ) {
+    this.stomp = new RxStomp();
+    this.router.afterEach(() => this.sendActivity());
+
+    this.store.watch(
+      (_state, getters) => getters.authenticated,
+      (value, oldValue) => {
+        console.log('TrackerService state change', value, oldValue);
```
Comment (qmonmert) [inline generators/client/templates/vue/src/main/webapp/app/admin/tracker/tracker.service.ts.ejs:28]: we keep that?

File: generators/client/templates/vue/src/main/webapp/app/shared/config/store/account-store.ts.ejs [hunk 2]
```diff
@@ -1,39 +1,49 @@
...
-        authenticated: false,
-        ribbonOnProfiles: '',
-        activeProfiles: '',
+export interface AccountStateStorable {
+  logon: boolean;
+  userIdentity: null | any;
+  authenticated: boolean;
+  ribbonOnProfiles: string;
+  activeProfiles: string;
+}
```
Comment (qmonmert) [inline generators/client/templates/vue/src/main/webapp/app/shared/config/store/account-store.ts.ejs:9]:
``` diff
+ };
- }
```

Comment (mshima) [reply generators/client/templates/vue/src/main/webapp/app/shared/config/store/account-store.ts.ejs:9]:
```suggestion
};
```

File: generators/client/templates/vue/src/main/webapp/app/admin/tracker/tracker.service.ts.ejs (hunk 1 above)
Comment (mshima) [reply generators/client/templates/vue/src/main/webapp/app/admin/tracker/tracker.service.ts.ejs:28]: It's somewhat useful, not sure.
Lets drop it.
```suggestion
```

**Review by qmonmert — APPROVED** [review]

**Pull request merged**